packages/angular_devkit/build_angular/test/browser/source-map_spec_large.ts

```diff
@@ -120,8 +121,10 @@ describe('Browser Builder source map', () => {
   it('supports styles only sourcemaps', (done) => {
     const overrides: Partial<BrowserBuilderSchema> = {
       sourceMap: {
```

Comment:
why are these options required now?

Reply:
They're not, I just went back and forth between required and not required.